lib/Conversion/AffineToPipeline/AffineToPipeline.cpp

```diff
@@ -255,6 +260,7 @@ LogicalResult AffineToPipeline::populateOperatorTypes(
   Problem::OperatorType mcOpr = problem.getOrInsertOperatorType("multicycle");
   problem.setLatency(mcOpr, 3);
 
+  SmallDenseMap<TypedValue<MemRefType> *, std::pair<unsigned, unsigned>> memOps;
```

Comment:
I don't see this being used, other than being populated during problem construction. Is it WIP or can it be removed? If you keep it, please explain what the numbers in the pair represent.